packages/openapi3/src/schema-emitter.ts

```diff
@@ -645,6 +651,63 @@ export class OpenAPI3SchemaEmitter extends TypeEmitter<
     }
   }
 
+  #attachXmlObject(program: Program, prop: ModelProperty, emitObject: OpenAPI3Schema) {
+    const xmlObject: OpenAPI3XmlSchema = {};
+
+    const isXmlModel =
+      prop.model && prop.model?.name !== resolveEncodedName(program, prop.model, "application/xml");
+
+    // Resolve XML name
+    const xmlName = resolveEncodedName(program, prop, "application/xml");
+    if (xmlName !== prop.name) {
+      xmlObject.name = xmlName;
+    }
+
+    // Get and set XML namespace if present
+    const currNs = getNs(program, prop);
+    if (currNs) {
+      xmlObject.prefix = currNs.prefix;
+      xmlObject.namespace = currNs.namespace;
+    }
+
+    // Set XML attribute if present
+    if (isAttribute(program, prop)) {
+      xmlObject.attribute = true;
+    }
+
+    // Handle array wrapping if necessary
+    if (
+      prop.type?.kind === "Model" &&
+      isArrayModelType(program, prop.type) &&
+      (xmlObject.name || isXmlModel)
+    ) {
+      xmlObject.wrapped = true;
+    }
+
+    // Set XML unwrapped if present
+    if (isUnwrapped(program, prop)) {
+      xmlObject.wrapped = false;
```

Comment:
```suggestion
     delete xmlObject.wrapped;
```
don't need to add it?